### PR TITLE
15286 - Jakarta mail 2.0 final release

### PR DESCRIPTION
**Pull request by uberskigeek**
#15286 
Update jakarta mail 2.0 to use final release jars

### CODE REVIEW

**Comment by uberskigeek** [comment]
#build

**Comment by uberskigeek** [comment]
#libbybot

**Comment by uberskigeek** [comment]
#build

**Comment by uberskigeek** [comment]
#build

**Comment by uberskigeek** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.mail-2.0.feature [hunk 1]
```diff
@@ -6,11 +6,9 @@ Subsystem-Version: 2.0
 IBM-Process-Types: client, \
  server
 -features=\
-  io.openliberty.jakarta.activation-2.0, \
- com.ibm.websphere.appserver.eeCompatible-9.0
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.mail-2.0.feature:10]: Why did you remove eeCompatible-9.0?

Comment (uberskigeek) [reply dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.mail-2.0.feature:10]: Trying to figure out why this feature is looking for the Jakarta mail implementation packages. It should not need them. Removed the eeCompatible-9.0 to see If that is the reason.

File: dev/com.ibm.websphere.appserver.features/visibility/public/mail-2.0/io.openliberty.mail-2.0.feature [hunk 2]
```diff
@@ -4,8 +4,9 @@ visibility=public
 singleton=true
 IBM-ShortName: mail-2.0
 Subsystem-Version: 2.0
-Subsystem-Name: mail 2.0
+Subsystem-Name: Jakarta mail 2.0
```
Comment (jhanders34) [inline dev/com.ibm.websphere.appserver.features/visibility/public/mail-2.0/io.openliberty.mail-2.0.feature:7]: This should conflict with my PR that I merged because It is `Jakarta Mail 2.0` instead of Jakarta mail 2.0`.

Comment (jhanders34) [reply dev/com.ibm.websphere.appserver.features/visibility/public/mail-2.0/io.openliberty.mail-2.0.feature:7]: It just may be easier to undo the subsystem name and rebase on integration

File: dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.mail-2.0.feature (hunk 1 above)
Comment (tevans78) [reply dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.mail-2.0.feature:10]: eeCompatible-9.0 isn't the cause of that, it doesn't bring in any code itself. Please put it back.

**Comment by uberskigeek** [comment]
#build

**Review by jhanders34 — APPROVED** [review]

**Comment by uberskigeek** [comment]
#libbyBot
